rotkehlchen/chain/ethereum/modules/liquity/decoder.py

```diff
@@ -125,22 +125,27 @@ def _decode_stability_pool_event(
                 event.event_subtype = HistoryEventSubType.DEPOSIT_ASSET
                 event.counterparty = CPT_LIQUITY
                 event.notes = f"Deposit {event.balance.amount} {self.lusd.symbol} in liquity's stability pool"  # noqa: E501
-            elif (
-                event.event_type == HistoryEventType.RECEIVE and
-                (
-                    event.asset == self.eth and
-                    event.balance.amount == collected_eth and
-                    tx_log.topics[0] == STABILITY_POOL_GAIN_WITHDRAW
-                ) or (
-                    event.asset == self.lqty and
-                    event.balance.amount == collected_lqty and
-                    tx_log.topics[0] == STABILITY_POOL_LQTY_PAID
-                )
-            ):
-                event.event_type = HistoryEventType.STAKING
-                event.event_subtype = HistoryEventSubType.REWARD
-                event.counterparty = CPT_LIQUITY
-                event.notes = f"Collect {event.balance.amount} {event.asset.resolve_to_crypto_asset().symbol} from liquity's stability pool"  # noqa: E501
+            elif event.event_type == HistoryEventType.RECEIVE:
+                if (
+                    (
+                        event.asset == self.eth and
+                        event.balance.amount == collected_eth and
+                        tx_log.topics[0] == STABILITY_POOL_GAIN_WITHDRAW
+                    ) or (
+                        event.asset == self.lqty and
+                        event.balance.amount == collected_lqty and
+                        tx_log.topics[0] == STABILITY_POOL_LQTY_PAID
+                    )
+                ):
+                    event.event_type = HistoryEventType.STAKING
+                    event.event_subtype = HistoryEventSubType.REWARD
+                    event.counterparty = CPT_LIQUITY
+                    event.notes = f"Collect {event.balance.amount} {event.asset.symbol_or_name()} from liquity's stability pool"  # noqa: E501
```

Comment:
Why symbol_or_name?

Reply:
In this case it should always use the symbol but alexey suggested it and I saw it cleaner than do the resolve and then call symbol, is what happens in that function